test/images/resource-consumer/consume-cpu/consume_cpu.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Can you add a comment for humans here for what system understands this magic string?

Reply:
this is a fairly standard [Go build tag](https://golang.org/pkg/go/build/#hdr-Build_Constraints), so I'm not sure that additional comments are necessary here.

Reply:
TIL. ACK